### ISSUE TITLE
Ability to automerge tags with the same slug

### ISSUE THREAD

**Issue by hroncok** [jazzband/django-taggit#218]
Currently, when the tag 'foo 1' is created, it has the slug 'foo-1'. Next, when tag 'foo-1' is created, it gets slug 'foo-1_1'.

I would like to merge the tags into one instead, using the name of the first used. I.e. instead of creating new tag 'foo-1', use the old one 'foo 1'.

Is that somehow possible?

**Comment by jsmedmar** [comment]
Hey, I think this is quite relevant. New tags shouldn't be created for matching slugs.

**Comment by jsmedmar** [comment]
This seems quite important!

**Comment by rtpg** [comment]
I don't agree that slug unification implies tag unification _for most cases_. That being said, there are cases where it totally does! I think we might want to add functionality to allow people to define the sort of "slug equivalency class"-style thing somehow here... or some other way to handle this (for example if you normalize the tags when getting data then this feature will happen automatically)

Definitely a valid enhancement proposal here.
